desktop/src/main/java/bisq/desktop/main/support/SupportView.java

```diff
@@ -112,11 +115,11 @@ public SupportView(CachingViewLoader viewLoader,
     public void initialize() {
         tradersMediationDisputesTab = new Tab();
         tradersMediationDisputesTab.setClosable(false);
-        //root.getTabs().add(tradersMediationDisputesTab); // hidden since mediation and refunds are not used in haveno
+        root.getTabs().add(tradersMediationDisputesTab);
 
         tradersRefundDisputesTab = new Tab();
         tradersRefundDisputesTab.setClosable(false);
-        //root.getTabs().add(tradersRefundDisputesTab);
+        root.getTabs().add(tradersRefundDisputesTab);
```

Comment:
Revert.

Reply:
As it appears in my workspace (unchanged) below:
![Screenshot from 2023-02-28 16-11-24](https://user-images.githubusercontent.com/20237127/221895329-54613ac6-fb58-4ff9-8a0c-63d25f5fff41.png)

Reply:
This still needs reverted. It should be commented out, same as master.

It was changed in this commit in your PR: https://github.com/haveno-dex/haveno/pull/545/commits/7bd72febdbee1bc41e4b84be7f267164f2d5a746